docs/using-flatpak.rst

```diff
@@ -8,24 +10,37 @@ commands.
 End users shouldn't generally need to use this page or the Flatpak command
 line interface, since Flatpak can be easily used through graphical software
 management tools, though they are of course free to use the command line if
-they prefer!
+they prefer! GNOME Software is one such example of a graphical application 
+to manage installing flatpaks.
 
 
 The flatpak command
 --------------------
 
 ``flatpak`` is the primary Flatpak command, to which specific commands are
 appended. For example, the command to install something is ``flatpak install``
-and the command to uninstall is ``flatpak uninstall``.
+and the command to uninstall is ``flatpak uninstall``. Additional subcommands 
+can be listed with ``flatpak --help``.
 
 Identifiers
 -----------
 
-Flatpak identifies each application and runtime using a unique three-part
-identifier, such as ``com.company.App``. The final segment of this address is
-the object's name, and the preceding part identifies the developer, so that
+Flatpak identifies each project (object) using a (three or more)-part identifier (ID),
+in reverse-DNS style. The final segment of this identifier is the project's name 
+the preceding segment typically identifies the developer so that
```

Comment:
I feel like this is over complicating things. Calling it always three part was indeed wrong though. Really its two part. A prefix which describes the organization that controls it and the application.

Some examples:

`org.gnome.Epiphany`
`org.gnome.Epiphany.Devel` (The application identifier is two segments)
`io.github.username.Application` (the organization is three segments)

We avoid `com.github` as that is, in theory, reserved for official Github software. Github uses `github.io` to host user websites.

Reply:
I swapped out the section in favor of the following, what do you think?


```
Flatpak identifies each project (object) using a two-part identifier (ID),
in reverse-DNS style. The final part of this identifier is the application's name.
The preceding segments identify the developer so that the same developer can have 
multiple applications, like ``com.company.App1`` and ``com.company.App2``. 
Some examples are:: 

``org.gnome.Epiphany`` (The first two segments are the organization, the final segment is the application)
``io.github.username.Application`` (the organization is the first three segments)

Reply:
On second thought, `io.github` could be swapped for `com.github`, as not too many (or any at all?) applications don't use `io.github` as the organization part of the ID.